packages/web/src/components/premium-content-purchase-modal/PremiumContentPurchaseModal.tsx

```diff
@@ -115,12 +112,16 @@ export const PremiumContentPurchaseModal = () => {
     }
   }, [stage, track, dispatch])
 
+  const mobile = isMobile()
+
   return (
-    <Modal
+    <ModalDrawer
```

Comment:
errr, should we not conditionally use `ModalDrawer` on mobile?
Something like 
```ts
const ModalComponent = mobile ? ModalDrawer : Modal

return (
  <ModalComponent ...
)
```

Reply:
ModalDrawer does all of this internally!